src/components/value/index.test.tsx

```diff
@@ -51,6 +60,11 @@ const inputOptions = {
   type: "url",
 };
 
+const mockDatasetWithBdayResourceInfo = SolidFns.setThing(
+  SolidFns.createSolidDataset() as any,
```

Comment:
Shouldn't be necessary I think?

```suggestion
  SolidFns.createSolidDataset(),
```